sky/clouds/azure.py

```diff
@@ -315,6 +314,10 @@ def make_deploy_resources_variables(
             'image_version': version,
         }
 
+        # Setup the A10 nvidia driver.
+        need_nvidia_driver_extension = (resources.accelerators is not None and
+                                        'A10' in resources.accelerators)
```

Comment:
`resources.accelerators` may not include the accelerators as we may have removed the `accelerators` field once the instance type is set?
```suggestion
        need_nvidia_driver_extension = (acc_dict is not None and
                                        'A10' in acc_dict)
```

Reply:
Good point! Done.